Kitodo-API/src/main/java/org/kitodo/api/MetadataGroup.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api;
+
+import java.util.List;
+
+public class MetadataGroup extends Metadata{
+
+    //The value of the metadata is a list of metadatas.
```

Comment:
a list of metadata objects

Reply:
It should be "metadata objects" instead of "metadata", correct?

Reply:
Yes.

Reply:
According to [https://en.oxforddictionaries.com/definition/data](url) "data" is not countable. So it should be a "a list of metadata objects"